media-gfx/nomacs/nomacs-3.14.2.ebuild

```diff
@@ -0,0 +1,92 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake xdg-utils
+
+PLUGIN_PKG="${PN}-plugins-$(ver_cut 1-2)"
+
+DESCRIPTION="Qt-based image viewer"
+HOMEPAGE="https://nomacs.org/"
+SRC_URI="
+	https://github.com/${PN}/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz
+	plugins? ( https://github.com/${PN}/${PN}-plugins/archive/$(ver_cut 1-2).tar.gz -> ${PLUGIN_PKG}.tar.gz )
+"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86 ~amd64-linux"
+IUSE="+jpeg +opencv plugins raw tiff zip"
+
+REQUIRED_USE="
+	raw? ( opencv )
+	tiff? ( opencv )
+"
+
+RDEPEND="
+	dev-qt/qtconcurrent:5
+	dev-qt/qtcore:5
+	dev-qt/qtgui:5[jpeg?]
+	dev-qt/qtnetwork:5
+	dev-qt/qtprintsupport:5
+	dev-qt/qtsvg:5
+	dev-qt/qtwidgets:5
+	media-gfx/exiv2:=
+	opencv? ( >=media-libs/opencv-3.4:= )
+	raw? ( media-libs/libraw:= )
+	tiff? (
+		dev-qt/qtimageformats:5
+		media-libs/tiff
+	)
+	zip? ( dev-libs/quazip )
+"
+DEPEND="${RDEPEND}"
+BDEPEND="
+	dev-qt/linguist-tools:5
+	virtual/pkgconfig
+"
+
+S="${WORKDIR}/${P}/ImageLounge"
+
+DOCS=( src/changelog.txt )
+
+src_unpack() {
+	unpack "${P}.tar.gz"
+	if use plugins ; then
+		unpack "${PLUGIN_PKG}.tar.gz"
+		mv "${PLUGIN_PKG}" "${S}/plugins" || die
+	fi
+}
+
+src_prepare() {
+	default
```

Comment:
do you need to run both `default` and `cmake_src_prepare`?

Reply:
`cmake_src_prepare` must be run instead of `default`.

Reply:
I conditionally modify .cmake file so it seems I need run cmake_src_prepare after applied changes, isn't it?

default is removed.

Reply:
no, cmake is not autotools ;)

I recommend to run `cmake_src_prepare` before modifying sources so that any patches continue to apply to vanilla source.

Reply:
Done,  `cmake_src_prepare` is moved to the top of `src_prepare()`